pkgs/misc/autotiling/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonApplication, fetchPypi, i3ipc, importlib-metadata }:
+
+buildPythonApplication rec {
+  pname = "autotiling";
+  version = "1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hwvy9bxwv9fakqqiyrkmpckxgm0z85c240p84ibdhja9sm086v0";
+  };
+
+  propagatedBuildInputs = [ i3ipc importlib-metadata ];
+  doCheck = false;
```

Comment:
Why aren't the checks working?

Reply:
wrapping '/nix/store/pmsr2q6r0cw67rhvnhh61azw61aky8fg-autotiling-1.1/bin/autotiling'...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
running install tests
no Makefile or custom buildPhase, doing nothing
pythonCatchConflictsPhase
pythonRemoveBinBytecodePhase
pythonImportsCheckPhase
Executing pythonImportsCheckPhase
setuptoolsCheckPhase
Executing setuptoolsCheckPhase
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
running egg_info
writing autotiling.egg-info/PKG-INFO
writing dependency_links to autotiling.egg-info/dependency_links.txt
writing entry points to autotiling.egg-info/entry_points.txt
writing requirements to autotiling.egg-info/requires.txt
writing top-level names to autotiling.egg-info/top_level.txt
reading manifest file 'autotiling.egg-info/SOURCES.txt'
writing manifest file 'autotiling.egg-info/SOURCES.txt'
running build_ext
usage: nix_run_setup
       [-h]
       [--debug]
nix_run_setup: error: unrecognized arguments: test
builder for '/nix/store/g4l6jjhc55lpk176zbkk8y5q1ks87y7j-autotiling-1.1.drv' failed with exit code 2

Reply:
found a solution just now https://github.com/NixOS/nixpkgs/pull/64529#discussion_r326165509

checkPhase = ''
    ${python.interpreter} test.py
'';

Reply:
I looked through the project and there appear to be no tests.

Reply:
okay, i thought that nix had its own tests. in the future i will mention it in the description if there are no tests.